CHANGELOG.md

```diff
@@ -35,6 +35,9 @@ Line wrap the file at 100 chars.                                              Th
   multihop, quantum-resistant tunnels, or DAITA.
 - Improved output format of `mullvad status` command, which now also prints feature indicators.
 
+#### Windows
+- Enable quantum resistant tunnels by default (when set to `auto`).
```

Comment:
Just a friendly note, I see both "quantum resistant" and "quantum-resistant" used,
but perhaps there is one format you prefer (internally at Mullvad). :v:

Reply:
Thanks, I've made the changelog consistent.